Assets/LeapMotion/Scripts/DataStructures/RingBuffer.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RingBuffer<T> {
+
+  private T[] arr;
+  private int firstIdx = 0;
+  private int lastIdx = 0;
+
+  public RingBuffer(int bufferSize) {
+    arr = new T[bufferSize];
+  }
+
+  public int Length {
```

Comment:
You have a getter here that uses UpperCamelCase but isFull uses lowerCamelCase :(